docs/UserGuide.md

```diff
@@ -266,6 +266,12 @@ Format: `add -e d/DESCRIPTION at/DATE_TIME [ap/CONTACT_INDEX_LIST]`
 
 DATE_TIME formats currently accepted
 * dd-MM-yyyy HH:mm
+* yyyy-dd-MM HH:mm
+* The following symbols can be used interchangeably to indicate date, time, year of the event.
+    * [.] (full stop)
+    * [/] (slash)
+    * [-] (hyphen)
+    * [:] (colon)
```

Comment:
Can people use a mix of these symbols? for example 2020.11/12 23-00? if they can maybe show an actual example, I think it will be even clearer than it is now

Reply:
Thanks for the suggestion. Have included the example below.